<ui version="4.0" >
 <class>PalssonKeggConverterClass</class>
 <widget class="QDialog" name="PalssonKeggConverterClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PalssonKeggConverter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Compounds dictionary</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLineEdit" name="lineEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pushButton" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>select file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="lineEdit_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Parsed compounds</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="lineEdit_3" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="5" row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Reaction mapping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Output:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Reactions input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLineEdit" name="lineEdit_4" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pushButton_2" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>select file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="lineEdit_5" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Parsed reactions</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="lineEdit_6" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="pushButton_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>FBA data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLineEdit" name="lineEdit_7" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pushButton_4" >
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>select file</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QLineEdit" name="lineEdit_8" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Parsed items</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLineEdit" name="lineEdit_9" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QComboBox" name="comboBox" >
        <item>
         <property name="text" >
          <string>NONE</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>BOTH</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
